[2025-01-29T22:08:44.593+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T22:08:44.665+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-01-29T22:06:00+00:00 [queued]>
[2025-01-29T22:08:44.688+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_processing.schema_create scheduled__2025-01-29T22:06:00+00:00 [queued]>
[2025-01-29T22:08:44.689+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-29T22:08:44.729+0000] {taskinstance.py:2889} INFO - Executing <Task(SQLExecuteQueryOperator): schema_create> on 2025-01-29 22:06:00+00:00
[2025-01-29T22:08:44.753+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2068) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T22:08:44.757+0000] {standard_task_runner.py:72} INFO - Started process 2077 to run task
[2025-01-29T22:08:44.758+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_processing', 'schema_create', 'scheduled__2025-01-29T22:06:00+00:00', '--job-id', '503', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl.py', '--cfg-path', '/tmp/tmpf8o4k4tq']
[2025-01-29T22:08:44.763+0000] {standard_task_runner.py:105} INFO - Job 503: Subtask schema_create
[2025-01-29T22:08:45.296+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_processing.schema_create scheduled__2025-01-29T22:06:00+00:00 [running]> on host a664bcf281bf
[2025-01-29T22:08:45.580+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_processing' AIRFLOW_CTX_TASK_ID='schema_create' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T22:06:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-29T22:06:00+00:00'
[2025-01-29T22:08:45.582+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T22:08:45.628+0000] {sql.py:278} INFO - Executing: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;
[2025-01-29T22:08:45.653+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-29T22:08:45.677+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-29T22:08:45.687+0000] {sql.py:544} INFO - Running statement: 
    create schema if not exists nds;
    create schema if not exists dds;
    create schema if not exists stg;, parameters: None
[2025-01-29T22:08:45.720+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T22:08:45.721+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_processing, task_id=schema_create, run_id=scheduled__2025-01-29T22:06:00+00:00, execution_date=20250129T220600, start_date=20250129T220844, end_date=20250129T220845
[2025-01-29T22:08:46.258+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T22:08:46.745+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T22:08:46.763+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
